### PR TITLE
Adjust link to FAQ

### LINKED ISSUE #2590: Broken link to FAQ in /hacs/entry
### Describe the issue

Visit /hacs/entry
Click on the FAQ link

Expected:
I'm taken to an FAQ; such as https://hacs.xyz/docs/faq/addons

Actual:
404 - https://hacs.xyz/docs/faq/what

### Screenshots

image

image

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fixes #2590

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
It does not make sense to go to /addons here.

**Comment by CloCkWeRX** [comment]
So, I put /addons here as it then mirrors the homepage:
image

Happy to adjust to whatever is best, but might also need to adjust the homepage behaviour too.

**Comment by ludeeus** [comment]
Website was wrong, adjusted here <https://github.com/hacs/documentation/commit/d0972e9ffcfa9e997a07ff1f1190d617d4a81c08>

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ HACS is a integration that gives the user a powerful UI to handle downloads of c
 
 - [General documentation](https://hacs.xyz/)
 - [Configuration](https://hacs.xyz/docs/configuration/basic)
-- [FAQ](https://hacs.xyz/docs/faq/what)
+- [FAQ](https://hacs.xyz/docs/faq/addons)
```
Comment (ludeeus) [inline README.md:27]:
```suggestion
- [FAQ](https://hacs.xyz/docs/faq)
```

File: info.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ https://github.com/hacs/integration/releases/tag/1.0.0
 
 - [General documentation](https://hacs.xyz/)
 - [Configuration](https://hacs.xyz/docs/configuration/basic)
-- [FAQ](https://hacs.xyz/docs/faq/what)
+- [FAQ](https://hacs.xyz/docs/faq/addons)
```
Comment (ludeeus) [inline info.md:19]:
```suggestion
- [FAQ](https://hacs.xyz/docs/faq)
```

**Comment by CloCkWeRX** [comment]
Nice. Good to go I think!

**Review by ludeeus — APPROVED** [review]

**Pull request merged**